src/abstractdataframe/abstractdataframe.jl

```diff
@@ -587,22 +590,40 @@ function nonunique(df::AbstractDataFrame)
     res
 end
 
+function nonunique(df::AbstractDataFrame, columns::Any)
+    nonunique(df[columns])
+end
+
+function nonunique(df::AbstractDataFrame, columns::Int)
```

Comment:
Because otherwise `unique(df, 2)` fails (must be `unique([2])` which I find less intuitive). I could do `Union{Int, Symbol} ` if that is preferred?